### PR TITLE
Ep/min 169 total collateral rpc

### PR DESCRIPTION
**Pull request by Apollo50**
**Description:**

Add RPC that allows to get user collateral in USD

**Changes in storage or API:**

Describe storage structure changes or API changes. Make sure to update the [API docs](https://minterestfinance.atlassian.net/wiki/spaces/MINTEREST/pages/134807557/Minterest+API).

**Checklist**

- [x] Code is covered with tests
- [ ] Code is covered with benchmarks
- [ ] Fresh benchmark results included

### CODE REVIEW

File: pallets/controller/src/lib.rs [hunk 1]
```diff
@@ -719,6 +719,40 @@ impl<T: Config> Pallet<T> {
 				)?;
 		Ok((total_supply_balance, total_borrowed_balance))
 	}
+
+	pub fn get_user_total_collateral(who: T::AccountId) -> BalanceResult {
```
Comment (lut1k) [inline pallets/controller/src/lib.rs:723]: I think, need to add comments for the function.

File: pallets/controller/rpc/src/lib.rs [hunk 2]
```diff
@@ -127,4 +130,20 @@ where
...
+		account_id: AccountId,
+		at: Option<<Block as BlockT>::Hash>,
+	) -> Result<Option<BalanceInfo>> {
+		let api = self.client.runtime_api();
+		let at = BlockId::hash(at.unwrap_or_else(||
+			// If the block hash is not supplied assume the best block.
+			self.client.info().best_hash));
+		api.get_user_total_collateral(&at, account_id).map_err(|e| RpcError {
+			code: ErrorCode::ServerError(Error::RuntimeError.into()),
+			message: "Unable to check if is an admin.".into(),
```
Comment (dev-vadim) [inline pallets/controller/rpc/src/lib.rs:145]: Wrong error message

**Review by lut1k — DISMISSED** [review]

File: pallets/controller/src/lib.rs (hunk 1 above)
Comment (Apollo50) [reply pallets/controller/src/lib.rs:725]: fixed

File: pallets/controller/rpc/src/lib.rs (hunk 2 above)
Comment (Apollo50) [reply pallets/controller/rpc/src/lib.rs]: fixed

**Review by lut1k — APPROVED** [review]

**Review by dev-vadim — APPROVED** [review]

**Review by kurotychb — APPROVED** [review]

**Pull request merged**